vars/ploigosWorkflowEverything.groovy

```diff
@@ -247,6 +247,8 @@ def call(Map paramsMap) {
     /* Name of the virtual environment to set up in the given home worksapce. */
     String WORKFLOW_WORKER_VENV_NAME = 'venv-ploigos'
 
+    String WORKFLOW_WORKER_VENV_PATH = "${WORKFLOW_WORKER_WORKSPACE_HOME_PATH}/${WORKFLOW_WORKER_VENV_NAME}"
```

Comment:
nit pick: can you add comment to this variable with like
```
/*path to veritual python environment that PSR is in and/or will be installed into, must be on a persistent volume that can be shared between containers */
```
or someting like that